metrics/src/metadata.rs

```diff
@@ -0,0 +1,46 @@
+#![allow(missing_docs)]
```

Comment:
Obviously this was in place for the draft aspect, but just noting it that we'll need to drop this and add docs for all the new public types here.